kyuubi-ha/src/main/scala/org/apache/kyuubi/ha/HighAvailabilityConf.scala

```diff
@@ -91,22 +91,31 @@ object HighAvailabilityConf {
       .checkValues(AuthTypes.values.map(_.toString))
       .createWithDefault(AuthTypes.NONE.toString)
 
+  val HA_ZK_AUTH_SERVER_PRINCIPAL: OptionalConfigEntry[String] =
```

Comment:
Simply for replacing kyuubi.ha.zookeeper.auth.principal?

Reply:
Not exactly.

For Kerberos authentication, it requires

- client principal
- client keytab
- server principal